orderer/common/follower/follower_chain.go

```diff
@@ -335,7 +351,7 @@ func (c *Chain) pull() error {
 	// Trigger creation of a new consensus.Chain.
 	c.logger.Info("Block pulling finished successfully, invoking chain creation callback")
```

Comment:
Nit: also here, shouldn't we update this log message to not reference the chain creation callback?

Reply:
done